### PR TITLE
ci(v8 release): migrate v8 docsite release to yaml

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Previous Behavior
- v8 docsite release is on a [separate](https://dev.azure.com/uifabric/UI%20Fabric/_release?_a=releases&view=mine&definitionId=22) release flow on ADO that triggers after v8 publish pipeline completes.

## New Behavior
- migrates v8 docsite release tasks to yaml as part of the v8 publish pipeline to allow it to be governed under the 1ESPT
- successful execution of publish docsite steps) can be found on this [pipeline run](https://dev.azure.com/uifabric/UI%20Fabric/_build/results?buildId=360306&view=logs&j=7494ae37-3ecd-50c6-9ffe-c77a2768c545&t=5bb77429-654b-500e-e3d8-426e1d44a450)
- adds `skipPublish` variable to v8 release pipeline to allow for docsite only updates if needed

### CODE REVIEW

File: .devops/templates/publish-website.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      scriptType: 'bash'
+      scriptLocation: inlineScript
+      inlineScript: |
+        az storage blob upload \
+        --account-name fabricweb \
+        --container-name fabric-website \
+        --name manifests/v${{ parameters.version }}-df.js \
+        --file $(System.DefaultWorkingDirectory)/site-manifests/v${{ parameters.version }}-df.js \
+        --auth-mode login \
+        --overwrite
```
Comment (TristanWatanabe) [inline .devops/templates/publish-website.yml:35]: This step updates the manifest file for the [staging docsite](https://developer.microsoft-tst.com/fluentui) which hasn't worked in a **very** long time. I just added it here for now to have a complete step by step migration of the release tasks but open to just removing it completely

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Review by tudorpopams — APPROVED** [review]

**Comment by tudorpopams** [comment]
Can you provide a bit more context on what "migrate [..] to yaml" means?

**Comment by TristanWatanabe** [comment]
@tudorpopams the original v8 docsite release pipeline is a "classic" pipeline meaning the whole pipeline was created through the ADO web portal and thus all jobs/steps are defined in ADO only. Migrating to yaml in this case brings in those pipeline jobs/steps into a `.yml` file so they can live in our codebase instead. This will also resolve a security related ticket.

**Pull request merged**